### ISSUE TITLE
Not able to pull image

### ISSUE THREAD

**Issue by admiralAwkbar** [assignUser/chktex-alpine#2]
Hey @assignUser 

Seems your docker image is blocked?
tried to pull the image from ghcr.io and got the following...

```bash
=> ERROR [internal] load metadata for ghcr.io/assignuser/chktex-alpine:0.1.1                                                                                                                                  0.4s
 => CANCELED [internal] load metadata for docker.io/garethr/kubeval:0.15.0                                                                                                                                     0.5s
 => CANCELED [internal] load metadata for docker.io/library/python:3.9-alpine                                                                                                                                  0.5s
------
 > [internal] load metadata for ghcr.io/assignuser/chktex-alpine:0.1.1:
------
failed to solve with frontend dockerfile.v0: failed to create LLB definition: failed to authorize: rpc error: code = Unknown desc = failed to fetch oauth token: unexpected status: 403 Forbidden
```

**Comment by assignUser** [comment]
Hey,

I did not change anything permission-wise. But it looks like there are some new package settings regarding access?
I have activated inherited access, so it should work now (as the repo is public)?
![image](https://user-images.githubusercontent.com/16141871/118519749-aee20900-b739-11eb-97d4-9ff0156a6530.png)

edit: I successfully tested pulling it locally but I am not sure in how far the git credential manager might influence things^^
